app/controllers/application_controller.rb

```diff
@@ -22,12 +23,11 @@ def append_to_log(params)
   # Rails' instrumentation code, and is run after each request.
   def append_info_to_payload(payload)
     super
-    if @custom_log_items
-      payload[:custom_log_items] = @custom_log_items
-    end
-  end
 
-  # :nocov:
+    append_to_log(request_id: RequestStore.store[:request_id])
```

Comment:
I disagree. I created the `append_to_log` so that you could append data to the log at any point in the request, rather than only in the magic `append_info_to_payload` method, which I consider bad design. By putting the `append_to_log` call in here it's suggesting a coupling which does not exist. There's already a before action to capture the request id, so I think it's cleaner to append to the log there.